server/sandbox.go

```diff
@@ -60,7 +65,7 @@ type podSandboxRequest interface {
 func (s *Server) getPodSandboxFromRequest(req podSandboxRequest) (*sandbox, error) {
 	sbID := req.GetPodSandboxId()
 	if sbID == "" {
-		return nil, fmt.Errorf("PodSandboxId should not be empty")
+		return nil, errSandboxIDEmpty
```

Comment:
@xlgao-zju We return a named error so that we can check for a specific error in sandbox_remove.go:

```go
sb, err := s.getPodSandboxFromRequest(req)
if err != nil {
	if err == errSandboxIDEmpty {
		return nil, err
	}
```